docs/tables/googleworkspace_gmail_message.md

```diff
@@ -21,7 +23,8 @@ from
   googleworkspace_gmail_message
 where
   user_id = 'user@domain.com'
-order by internal_date
+  and query = 'newer_than:7d'
```

Comment:
@judell Do we need the `query` in this query, since we're already limiting it 10 items, which gets passed into the query params, so we're getting 10 items at most from the API? We have an example below that uses the `newer_than` query string as well.

Was your intention to limit the number of messages, or change the types of messages this query returned?

Reply:
This was the first query I tried, and it failed with Quota Exceeded. For a mailbox with substantial history, the query as written only avoids that if you only ask for `id` and/or `thread_id`. This might be an issue to raise in the fdw repo? Anyway this is why I've also added the bit about almost always needing a `query=` filter. Without it, and altering `MaxConcurrency` as I mentioned elsewhere, I was able to pull 138K messages into a matview. But that's something you'd only do for research purposes, not casual querying.